sql/histograms/cruxDcl.sql

```diff
@@ -0,0 +1,46 @@
+#standardSQL
+CREATE TEMPORARY FUNCTION spreadBins(bins ARRAY<STRUCT<start INT64, `end` INT64, density FLOAT64>>)
```

Comment:
Tried unsuccessfully to include this as an external library:

```sql
CREATE TEMPORARY FUNCTION spreadBins(bins ARRAY<STRUCT<start INT64, `end` INT64, density FLOAT64>>)
RETURNS ARRAY<STRUCT<client STRING, start INT64, density FLOAT64>>
LANGUAGE js AS "return spreadBins(bins);";

OPTIONS (
  library="gs://httparchive/reports/udf/crux.js"
);
```

Here's the lib: https://cdn.httparchive.org/reports/udf/crux.js

Keep getting the error `CREATE TEMPORARY FUNCTION statements must be followed by an actual query.`

Thought maybe the function names can't be duplicates but got same error. Otherwise it looks just like the example.

Reply:
Paging @fhoffa to the rescue :)